### PR TITLE
Add spanish translation for-misc

### PR DESCRIPTION
**Pull request by BlueeeMoon**

### CODE REVIEW

File: bitcoin-core-dev-tech/2022-10-10-misc.es.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+¿Podrías encontrar una curva que fuera más fácil de hacer a mano? Quizá una curva binaria sobre un campo binario. El campo base en vez de ser un primo grande sería un campo de Galois con `2^256` elementos o algo así. No sé si las operaciones son más fáciles de hacer allí. Supongo que a los humanos les resultaría más fácil operar en base `10`. El material existente que tiene Andrew funciona sobre el campo de `32` elementos. Pero entonces hay maquinaria extra con la que trabajar...
+
+Sugerí hace mucho tiempo, ni idea de lo factible que es, utilizar ayudas mecánicas. Hay ordenadores que pueden evaluar las transformadas de Fourier. Hay algoritmos de multiplicación que se basan en transformadas de Fourier. Como máquinas de cuentas o hardware mecánico. Algo así tal vez podría hacerse funcionar. Es cuestión de tiempo. Evaluar una multiplicación escalar de un punto de curva implica algo así como mil aproximaciones de Fourier... cada punto es `10`, así que hay `100` de ellos, así que hay alrededor de `1.000`.
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2022-10-10-misc.es.md:34]: I think is unnecessary to use the `` syntax for the numbers in the above 2 paragraphs.

File: bitcoin-core-dev-tech/2022-10-10-misc.es.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+El verdadero 0conf debería ser "sólo envía la clave privada". Eso es incluso más rápido que 0conf. Me pregunto por qué ese no fue el punto de esquematización en torno al cual la gente se organizó.
+
+# Redes de tráfico anónimo
+
+Anonimizador de paquetes usando un HSM: la privacidad del tráfico está protegida por la raíz de confianza del HSM. Utilizar un HSM para enrutar el tráfico anónimo. Esto podría diseñarse de forma que sea más fácil detectar si alguien está manipulando una red, o crear un incentivo o presión para hacer raíces de confianza más fuertemente protegidas, como raíces de confianza descentralizadas acordadas por grupos.
+
+# Firma a mano
+
+Una idea es cifrar/descifrar a mano y exportar el proceso de firma a un ordenador utilizando un cifrado totalmente homomórfico. Podrías sustituir el algoritmo de firma por una versión totalmente homomórfica del algoritmo de firma. O si se trata principalmente de cálculos de celosía, entonces puede ser posible hacerlo a mano. Digamos que obtienes un blob encriptado completamente homomórfico y entonces sólo hay un algoritmo que puede ejecutarse en él para escupir `0` o `1` para la verificación.
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2022-10-10-misc.es.md:28]: `0` and `1` do not need the `` syntax.

File: bitcoin-core-dev-tech/2022-10-10-misc.es.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Ayer hablábamos de esto. Podrías precalcular... podrías tener tablas de cálculo para todas las potencias ya escritas en un libro y tal vez podrías hacer las cosas más eficientemente de esa manera. Parece que sería fácil poner pequeños errores que resultarían en valores erróneos que filtrarían información.
+
+El mejor caso sería como una comprobación de emparejamiento. ¿Qué tan difícil es hacer un cálculo de emparejamiento a mano? Es mucho más caro que hacer una simple suma de puntos, ¿pero quizás es mejor que una multiplicación de puntos? La parte cara de calcular un emparejamiento es que el emparejamiento de `BLS-12`, por ejemplo, entra en un campo de tamaño `p^12`, así que hay un valor r de unos `256` bits que divide `p^12 - 1`. Así que hay que multiplicar `p^12 - 1` dividido por `r`, que es un valor de aproximadamente `2000` bits o algo así. Es una gran exponenciación.
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2022-10-10-misc.es.md:38]:
```suggestion
El mejor caso sería como una comprobación de emparejamiento. ¿Qué tan difícil es hacer un cálculo de emparejamiento a mano? Es mucho más caro que hacer una simple suma de puntos, ¿pero quizás es mejor que una multiplicación de puntos? La parte cara de calcular un emparejamiento es que el emparejamiento de BLS-12, por ejemplo, entra en un campo de tamaño `p^12`, así que hay un valor `r` de unos 256 bits que divide `p^12 - 1`. Así que hay que multiplicar `p^12 - 1` dividido por `r`, que es un valor de aproximadamente 2000 bits o algo así. Es una gran exponenciación.
```

File: bitcoin-core-dev-tech/2022-10-10-misc.es.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+La verificación de Bulletproofs a mano sería como Sysphus empujando un bolder a mano. ¿Hacer multiplicaciones de `1000` puntos a mano? Cometes un pequeño error y se acabó. Si cometes un pequeño error, ¿todavía tienes un punto en la curva? Si cometes un pequeño error, depende de cómo lo hagas. ¿Un error aleatorio? Si eliges una `x` al azar, hay un 50% de posibilidades de que sea un punto de la curva. ¿Pero una `x` y una y aleatorias que tú elijas? Lo más probable es que no sea un punto de curva.
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2022-10-10-misc.es.md:40]:
```suggestion
La verificación de Bulletproofs a mano sería como Sysphus empujando un bolder a mano. ¿Hacer multiplicaciones de 1000 puntos a mano? Cometes un pequeño error y se acabó. Si cometes un pequeño error, ¿todavía tienes un punto en la curva? Si cometes un pequeño error, depende de cómo lo hagas. ¿Un error aleatorio? Si eliges una `x` al azar, hay un 50% de posibilidades de que sea un punto de la curva. ¿Pero una `x` y una `y` aleatorias que tú elijas? Lo más probable es que no sea un punto de curva.
```

**Review by kouloumos — COMMENTED** [review]
I appreciate the fact that you are putting extra effort to use the `` syntax since [I pointed that out](https://github.com/bitcointranscripts/bitcointranscripts/pull/172#discussion_r914609328) a few months ago, but I think that there are occasions that it's not appropriate and might even make the reading experience worst. This should be primarily used on equations and when the speaker refers to symbols in an equation. I pointed out some places that I think should be removed, and others that should be added:

**Comment by BlueeeMoon** [comment]
Ok, thanks, I've updated it.

**Pull request merged**